### ISSUE TITLE
Table widget for Tabulated trapezium, Tabulated rectangle and YZ shape cross sections

### ISSUE THREAD

**Issue by leendertvanwolfswinkel** [nens/threedi-schematisation-editor#90]
In the current implementation, the height-width pairs input for "Tabulated trapezium" and "Tabulated rectangle" shapes is given in a multiline text widget:

![image](https://user-images.githubusercontent.com/26224299/206197302-079d9d32-b7b8-4d90-b12b-adb4c637f022.png)

One of the new shapes we want to introduce (YZ) also needs table input, but for this shape (width, height) makes more sense. I feel that using the same multiline text widget for both of these makes it too complicated. Instead, a table widget would be more appropriate.

- [x] If the selected shape is "Tabulated trapezium" or "Tabulated rectangle", the table widget should have the 2 columns: Height [m] and Width [m].
- [x] If the selected shape is "YZ", the table widget should have the 2 columns: Y [m] and Z [m].- [ ] It should be easy to add a new row, e.g. by pressing Enter when in the bottom-right cell of the table
- [x] It should be possible to copy-paste table data from e.g. MS Excel into the widget
- [x] If the table has a lot of rows, it should become scrollable

**Comment by leendertvanwolfswinkel** [comment]
labelled as bug because since we moved to schema version 214, this will probably give errors if users use this type of profile

**Comment by leendertvanwolfswinkel** [comment]
Probably makes sense to fix #98 at the same time

**Comment by leendertvanwolfswinkel** [comment]
I tried the following:
- Add a cross section location
- Set its cross section shape to 'YZ'
- Paste this into the table: 
```
0,7.77
0.17,7.62
0.4,7.55
0.88,7.35
1.57,7.1
2.34,6.79
3.32,6.65
4.23,6.63
4.47,6.64
5.1,6.67
5.83,6.8
7.16,7.09
8.14,7.42
8.57,7.57
8.94,7.77
```
- Save to spatialite
- Load from spatialite
- Cross section location's table is empty

Also some minor issues:
- In the dropdown widget to select the cross section shape, please capitalize YZ (i.e., 'YZ' instead of 'Yz'
- Widgets that are enabled do not always have their labels enabled, e.g.: ![image.png](https://images.zenhubusercontent.com/62bedb7b667872a01ff3bdff/838e1f4d-6cb8-4518-83ff-b3eb0a4243f8)
- Inverted egg: 'height' should also be disabled
